asapdiscovery-cli/asapdiscovery/cli/cli_args.py

```diff
@@ -87,6 +87,15 @@ def output_dir(func):
     )(func)
 
 
+def overwrite(func):
+    return click.option(
+        "--overwrite",
+        is_flag=True,
+        default=False,
```

Comment:
not sure if this was ready for re-review but i'll just add here anyway with some info. Still need to change the default. You can use the `--overwrite/--no-overwrite` constructor for `click` that handles these kinds of things nicely.

eg:
```python
@click.option("--allow-dask-cuda/--no-allow-dask-cuda", default=True)
```

Reply:
It wasn't, but thank you! I will add that